spec_parser/main.py

```diff
@@ -3,12 +3,16 @@
 # SPDX-License-Identifier: Apache-2.0
 
 from spec_parser import Model
-from runparams import RunParams
+from spec_parser.runparams import RunParams
```

Comment:
why move it?  `RunParams` are for processing command-line options. Not really anything to do with the spec-parsing itself.

Reply:
This is to avoid namespace collisions when installing the parser in a Python environment.  If we're concerned about clarity surrounding purpose, we could move `main.py` and `runparams.py` into something like `spec_parser.cmd`.  But that might make it more inconvenient to run `main.py` directly.  Was there a reason we separated the run parameters into their own file if they have no use outside of the command line?